William Seemann <wseemann@gmail.com>

[PATCH 3/3] Fix ‘UINT64_C’ was not declared in this scope error
---

--- a/libavutil/common.h	2015-01-06 12:58:59.000000000 -0600
+++ b/libavutil/common.h	2015-01-20 01:45:16.000000000 -0600
@@ -26,6 +26,14 @@
 #ifndef AVUTIL_COMMON_H
 #define AVUTIL_COMMON_H
 
+#ifdef __cplusplus
+#   define __STDC_CONSTANT_MACROS
+#ifdef _STDINT_H
+#   undef _STDINT_H
+#endif
+#   include <stdint.h>
+#endif
+
 #if defined(__cplusplus) && !defined(__STDC_CONSTANT_MACROS) && !defined(UINT64_C)
 #error missing -D__STDC_CONSTANT_MACROS / #define __STDC_CONSTANT_MACROS
 #endif
